R/Features-functions.R

```diff
@@ -22,9 +22,6 @@ number_assays_in_se <- function(object) {
 
 .valid_assay_links <- function(object) {
     n_exp <- names(experiments(object))
-    n_alk <- names(object@assayLinks)
-    if (any(n_exp != n_alk))
```

Comment:
Could you clarify why you have removed this test. Is it because between adding an assay with `addAssay` and adding the assay link with `addAssayLink` (for example in `logTransform`), we end up with a temporarily invalid object?

Reply:
Indeed this test is very important, but it is already checked in the `.valid_Features_indices` and both `.valid_Features_indices` and `.valid_assay_links` are called by `.valid_Feature` for validity check. The question becomes: do we really need `.valid_Features_indices`? If not, we can put this check back in `.valid_assay_links`.

Reply:
I see, thanks. OK, let's keep it how you changed it.

I think there will be a need for a discussion about the validity functions at some point, especially in the light of #58 and #52